java/spark/v3/README.md

```diff
@@ -0,0 +1,79 @@
+# Spark-Lance Connector
```

Comment:
@eddyxu , do we have plans to add this documentation to our doc website later?

Reply:
Yes, we should do that